melange/init

```diff
@@ -10,6 +10,7 @@ mount -t sysfs sys -o nodev,nosuid,noexec /sys
 mount -t tmpfs -o nodev,nosuid tmpfs /tmp
 mkdir -p -m 0755 /dev/shm
 mount -t tmpfs -o nodev,nosuid,mode=1777 tmpfs /dev/shm
+mount -t cgroup -o all cgroup /sys/fs/cgroup
```

Comment:
please use these options and settings:

https://github.com/systemd/systemd/blob/df5dd05941da82b3ed291a85595098b168eabda8/src/shared/mount-setup.c#L109

please do not mount cgroup v1, only cgroup2

Reply:
```suggestion
mount -t cgroup2 -o nsdelegate,memory_recursiveprot,nosuid,nodev,noexec cgroup2 /sys/fs/cgroup
```